### ISSUE TITLE
CentOS 15.2.0-2248 RPM does not register with Artifactory as a valid RPM after upload

### ISSUE THREAD

**Issue by tedops** [gocd/gocd#1560]
After uploading the 15.2 RPM to Artifactory, the RPM fails to be read as a valid RPM. The message displayed is:

    Unable to find package info (broken RPM?).

Artifactory shows the correct filesize and other info, but the RPM info isn't recognized. 

This problem did NOT happen with the 15.1 RPM. Plus, I can see the contents and scripts without problems using ```rpm``` from the command line (e.g. ```rpm -qpil go-server-15.2.0-2248.noarch.rpm```)

Because of this issue, yum doesn't recognize the 15.2 RPM as a valid package. The result is that ```yum install``` and ```yum list --showduplicates``` commands fail to detect/install 15.2, despite it being available.

Were there any changes made to the packaging process for the go-server RPM? Perhaps there was a switch from manual spec file writing to using FPM, or vice versa?

Thanks very much in advance

**Comment by ketan** [comment]
Nothing major that could cause this to not work as expected has changed with our spec files (we don't use FPM, yet)

* https://github.com/gocd/gocd/commits/master/installers/agent/rpm/go-agent.spec
* https://github.com/gocd/gocd/commits/master/installers/server/rpm/go-server.spec

[speculation]Could this be because of lack of package signing?[/speculation]

**Comment by tedops** [comment]
@ketan Thanks for the super quick response. I just did further testing and found out that when the 15.2 RPM is uploaded through the Artifactory UI, the RPM info is just fine.

We're using a custom Python tool that uses the Artifactory REST API to upload files. This tool works fine on a daily basis for countless artifacts, but something is making the go-server 15.2 RPM behave strangely ("no RPM info") when it is uploaded using this tool.

Anyways, thanks again. Closing ticket.
